python/gui/auto_generated/processing/qgsprocessingalgorithmdialogbase.sip.in

```diff
@@ -116,6 +116,11 @@ the given ``format``.
 Reports an ``error`` string to the dialog's log.
 
 If ``fatalError`` is ``True``, the error prevented the algorithm from executing.
+%End
+
+    void pushWarning( const QString &warning );
+%Docstring
+Pushes a warning information string to the dialog's log.
```

Comment:
add \since

I think we should backport to 3.16, so tag with \since qgis 3.16.2

Reply:
I think we should too.